x-pack/plugins/monitoring/server/routes/api/v1/elasticsearch/ccr.js

```diff
@@ -139,6 +140,14 @@ function buildRequest(req, config, esIndexPattern) {
               terms: {
                 field: 'ccr_stats.leader_index',
                 size: 1
+              },
+              aggs: {
+                remote_cluster: {
+                  terms: {
+                    field: 'ccr_stats.remote_cluster',
+                    size: 1
+                  }
+                }
```

Comment:
This is probably for a later PR but:

I see that you are doing a `collapse` on `ccr_stats.follower_index` above. This means that there's a doc returned in the hits for every unique follower index. Given that every follower index can have exactly one `leader_index` and `remote_cluster`, I wonder if we need the `leader_index` and `remote_cluster` aggs here. We should be able to just lookup the `ccr_stats.leader_index` and `ccr_stats.remote_cluster` fields in each returned doc.